server/src/test/java/org/elasticsearch/cluster/coordination/PublicationTransportHandlerTests.java

```diff
@@ -137,7 +137,6 @@ private static boolean isDiff(BytesTransportRequest request, DiscoveryNode node)
                 if (compressor != null) {
                     in = new InputStreamStreamInput(compressor.threadLocalInputStream(in));
                 }
-                in.setTransportVersion(node.getVersion().transportVersion);
```

Comment:
While it's unlikely we will change the wire representation of a single boolean field, it's still a possibility. I think we should continue to be explicit about the transport version in use here.

Reply:
The issue is that I couldn't find a way to get a handle on the Connection at this point in teh test

Reply:
You'll need to pass the transport version in from the caller (who should know what transport version each node in the test is using)